batch_convert.qml

```diff
@@ -24,7 +24,7 @@ MuseScore {
         text: qsTr("This plugin needs MuseScore 3")
         onAccepted: {
             window.visible = false
```

Comment:
should no longer be required, as the window (now called pluginId) is closed by the next line

Reply:
Should that name be unique, would it conflict with other plugins?

Reply:
Better save than sorry, so I picked `batchConvert` as the id, thanks.

Reply:
The id of the main plugin object is unique within the plugin scope; it does not conflict with other plugin instantiations, fyi

Reply:
OK, thanks, got it